datamanager/splitLargeZip/splitZip.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+"""
+This script is used to split zip files (DM exports) that are above 1GB in size.
+
+Usage:
+    splitZip.py --path C:\\path\\to\\zip
+
+The result of running the script is multiple zip files that are all below 1GB.
+"""
+import argparse
+import csv
+import os
+import tempfile
+import zipfile
+from dataclasses import dataclass
+from io import TextIOWrapper
+from pathlib import Path
+
+ONEGB = 1000000000
+
+
+@dataclass
+class configs:
+    path_to_schema = ""
+    path_to_split = ""
+    start_size = 0  # size of schema and split file
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--path", help="the path of the zip")
+    args = parser.parse_args()
+    processZip(args.path)
+
+
+def processZip(path):
+    with tempfile.TemporaryDirectory() as temp_zip_contents:
+        with zipfile.ZipFile(path, "r") as zip_ref:
+            zip_ref.extractall(temp_zip_contents)
+            dictionary_paths = {
+                path.name: path for path in list(Path(temp_zip_contents + "/" + path.rstrip(".zip")).iterdir())
+            }
+
+            configs.path_to_schema = dictionary_paths["schema.json"]
+            configs.start_size += os.stat(configs.path_to_schema).st_size
```

Comment:
```suggestion
            configs.start_size += configs.path_to_schema.stat().st_size
```
if this is Path object